CONVENTIONS.md

```diff
@@ -126,6 +126,22 @@ it is used to manage all elements of the cluster.
 * Every component must remain available to consumers without disruption during upgrades and reconfiguration
   * See the next section for more details
 
+#### High Availability
+
+We focus on minimizing the impact of a failure of individual nodes in a cluster by ensuring operators or operands are spread across multiple nodes while maintaining backwards compatibility.
```

Comment:
What does "while maintaining backwards compatibility" have to do with these guidelines?  Is this just a reminder that having multiple pod replicas may necessitate extra precautions related to version skew?

Reply:
backwards compatibility refers to 2 worker nodes requirement supported.

Reply:
Are you saying there was a version of OpenShift where the minimum was 3 worker nodes, and the focus must include maintaining compatibility with those clusters?  Isn't that a vacuous requirement?  I mean, if these guidelines avoid breaking 2-worker clusters, is extra effort required not to break 3-worker clusters?

Reply:
Nope, we are concerned about 2-worker clusters which we expanded in the later section. In order to avoid confusion, I am dropping this line altogether.